src/vs/base/parts/quickinput/browser/quickInput.ts

```diff
@@ -695,6 +696,15 @@ class QuickPick<T extends IQuickPickItem> extends QuickInput implements IQuickPi
 		this.update();
 	}
 
+	get customButtonAriaLabel() {
+		return this._customButtonAriaLabel;
+	}
+
+	set customButtonAriaLabel(label: string | undefined) {
```

Comment:
who actually sets this value?

Reply:
Extensions who want to reuse / build on the quickpick, and would like to have separate aria-label on the custom button.

Reply:
This change will require API review so it's best to separate that commit out to another PR as there is some more plumbing that needs to be done for it to make it all the way to extensions.

Reply:
API change reverted for now.